packages/styles/src/components/tag.scss

```
@@ -0,0 +1,53 @@
+@forward './../variables/options';
+@forward './utilities';
+
+@use './../variables/components/tag' as tag;
+
+.tag:has(:not(pre)), // is only a temporary fix till I know how to do it wihtout causing future issues. wihtout this the Code sample would break currently.
+.tag-sm {
+  display: inline-flex;
+  justify-content: flex-start;
+  justify-items: center;
+  align-items: center;
+  gap: tag.$tag-gap;
+  padding: tag.$tag-vertical-padding tag.$tag-horizontal-padding;
+  font-size: tag.$tag-font-size;
+  border: tag.$tag-border-width solid transparent;
+  border-radius: tag.$tag-border-radius;
+  line-height: inherit;
+  text-align: center;
+  vertical-align: baseline;
+  max-width: tag.$tag-max-width;
+
+  &.bg-white {
+    border-color: tag.$tag-white-border-color;
+  }
+
+  .tag-content {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+}
+
+.tag:has(:not(pre)) {
+  height: tag.$tag-height;
+
+  .tag-icon {
```

This additional class is not necessary, you can simply select the `post-icon` with its tag name:
```suggestion
  post-icon {
```